velox/exec/HashBuild.cpp

```diff
@@ -312,6 +312,7 @@ void HashBuild::removeInputRowsForAntiJoinFilter() {
 }
 
 void HashBuild::addInput(RowVectorPtr input) {
+  traceInput(input);
```

Comment:
this could be trace as well? Need similar processing. We shall have a trace collection test when hash build and probe spill. Thanks!